elasticjob-infra/elasticjob-restful/src/test/java/org/apache/shardingsphere/elasticjob/restful/pipeline/HttpClient.java

```diff
@@ -66,8 +65,11 @@ protected void initChannel(final Channel ch) throws Exception {
                                 .addLast(new SimpleChannelInboundHandler<FullHttpResponse>() {
                                     @Override
                                     protected void channelRead0(final ChannelHandlerContext ctx, final FullHttpResponse httpResponse) throws Exception {
-                                        consumer.accept(httpResponse);
-                                        countDownLatch.countDown();
+                                        try {
+                                            consumer.accept(httpResponse);
+                                        } finally {
+                                            countDownLatch.countDown();
```

Comment:
Why need CountDownLatch? Maybe I have to clone the branch for deeply reviewing .

Reply:
> Why need CountDownLatch? Maybe I have to clone the branch for deeply reviewing .

The HTTP client for unit test is implemented via Netty, which handle http response asynchronously. So I use CountDownLatch to wait for http response consumer finish.